lib/elixir_tw/web/static/css/app.scss

```diff
@@ -0,0 +1,3 @@
+$icon-font-path: '/fonts/'; /* use fonts from priv/static/fonts/ */
```

Comment:
Use `//` comments everywhere

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#comment)